ecosystem/sep-0038.md

```diff
@@ -0,0 +1,200 @@
+}
+```
+
+### Asset Identification Format
+
+This protocol can be used to provide quotes for any class of asset in exchange for a Stellar asset. The following format is used to identify an asset in API requests and responses.
+
+```
+<scheme>:<identifer>
+```
+
+The currently accepted `scheme` values are:
+
+Name | Description
+-----|------------
+`stellar` | Used for Stellar assets.
+`iso4217` | Used for fiat currencies.
+
+`stellar` Identifier Format
+
+The [SEP-11](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0011.md#asset) asset format.
+
+`iso4217` Identifier Format
+
+The [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) code for the fiat currency.
+
+### Endpoints
+
+* [`GET /info`](#get-info)
+* [`GET /prices`](#get-prices)
+* [`GET /price`](#get-price)
+* [`POST /quote`](#post-quote)
+
+### GET Info
+
+This endpoint describes the supported Stellar assets and off-chain assets available for trading. Note that the anchor may not support a trading pair between every Stellar asset and off-chain asset listed. Use the [`GET /prices`](#get-prices) endpoint to see which pairs are supported.
+
+#### Request
+
+No request arguments required.
+
+#### Response
+
+Name | Type | Description
+-----|------|------------
+`assets` | array | An array of objects describing the assets available in exchange for one or more of the other assets listed.
+
+`assets` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`asset` | string | The [Asset Identification Format](#asset-identification-format) value.
+`delivery_methods` | array | (optional) Only for non-Stellar assets. An array of objects describing the methods a client can use to deliver funds to the anchor. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response.
+
+`delivery_methods` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`name` | string | The value to use when making `POST /quote` requests.
+`description` | string | A human readable description of the method identified by `name`.
+
+### GET Prices
+
+This endpoint can be used to fetch the [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) prices of available off-chain assets in exchange for a Stellar asset and vice versa.
+
+#### Request
+
+Name | Type | Description
+-----|------|------------
+`sell_asset` | string | The asset you want to sell, using the [Asset Identification Format](#asset-identification-format).
+
+#### Response
+
+Name | Type | Description
+-----|------|------------
+`buy_assets` | array | An array of objects containing information on the assets that can be exchanged for `sell_asset`.
+
+`buy_assets` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`asset` | string | The [Asset Identification Format](#asset-identification-format) value.
+`price` | string | The price offered by the anchor for one unit of `asset` in terms of `sell_asset`.
```

Comment:
💡 For those coming from a financial background that understand base and counter well, this description could be a good place to use it and to say which asset is the base and which is the counter. I'd still keep the description about which asset is one unit, etc.